sbi/analysis/plot.py

```diff
@@ -305,7 +311,9 @@ def pairplot(
         subset: List containing the dimensions to plot. E.g. subset=[1,3] will plot
             plot only the 1st and 3rd dimension but will discard the 0th and 2nd (and,
             if they exist, the 4th, 5th and so on).
-        upper: Plotting style for upper diagonal, {hist, scatter, contour, cond, None}.
+        offdiag: Plotting style for upper diagonal, {hist, scatter, contour, cond,
+            None}.
+        upper: deprecated, use offdiag instead.
```

Comment:
We have to remember to put this in the changelog.

Reply:
and you agree that this change is useful, e.g., that we should be consistent in calling it `offdiag` like the other `kwargs`?